### ISSUE TITLE
What

### ISSUE THREAD

**Issue by Herteby** [Akryum/vue-meteor-tracker#13]

**Comment by Akryum** [comment]
The data returned is immutable and you have to clone it to modify it.

**Comment by Herteby** [comment]
Ah, but when would you want to modify something returned by Meteor? Wouldn't that be bad practice anyway? Since it would just get overridden when Meteor updates again.
So I'll just keep it set to true then.

**Comment by Akryum** [comment]
Yes, that is what I would do.

**Comment by mitar** [comment]
But is this checked anywhere by Vue?

**Comment by Herteby** [comment]
It gets checked by vue-meteor-tracker:
https://github.com/meteor-vue/vue-meteor-tracker#freezing-data

**Comment by mitar** [comment]
No, I found it. There is a check `Object.isExtensible()` in Vue code. I was searching for `Vue.isFrozen`.

**Comment by Herteby** [comment]
Ah, I thought you meant the Vue.config.meteor.freeze option 😛

**Comment by dolle39** [comment]
Is it possible to only freeze for example one meteor subscription? Or perhaps only set 
`Vue.config.meteor.freeze = true;` for one Vue component and not assign it globally?

**Comment by Akryum** [comment]
You can freeze the data yourself in the `update` function.
